src/graphic/Fast3D/gfx_pc.cpp

```diff
@@ -3283,11 +3291,15 @@ bool gfx_set_fb_handler_custom(Gfx** cmd0) {
     return false;
 }
 
+float gfx_calculate_noise_scale() {
+    return ((float)gfx_current_game_window_viewport.height / SCREEN_HEIGHT) * 0.5f;
```

Comment:
My previous comment meant for this to be
```suggestion
    return ((float)gfx_current_dimensions.height / gfx_native_dimensions.height) * 0.5f;
```

So that it retains what you are replacing from before (specially that we don't use `SCREEN_HEIGHT` here anymore)